frappe/desk/doctype/tag/tag.py

```diff
@@ -81,7 +81,7 @@ def update(self, dn, tl):
 		if not tl:
 			tags = ''
 		else:
-			tl = list(set(filter(lambda x: x, tl)))
+			tl = list(dict.fromkeys(filter(lambda x: x, tl)))
```

Comment:
this only works if you use Python >= 3.7

Reply:
this works in python >= 2.7 as well